### ISSUE TITLE
Support copy png description metadata to avif description

### ISSUE THREAD

**Issue by Kieaer** [AOMediaCodec/libavif#1564]
Same title.
When converting png format to avif, png description metadata is lost.

**Comment by y-guyon** [comment]
Thank you for the report.

Is there a PNG file example that you can share?

Do you know if another format than AVIF keeps the description when converted from PNG?

**Comment by Kieaer** [comment]
@y-guyon I've never heard of copying descriptions between different formats.
Here's example png contains description in metadata
https://drive.google.com/file/d/11x84VJFjMdYI8M0Eb0lMqvj1e7YNiqcA/view?usp=sharing

**Comment by Kieaer** [comment]
@kmilos After downloading the Image from the Google Drive link above, you can check the PNG:Description at the site below.
https://exifmeta.com/

I already tried using Exiftool, but the only thing that didn't copy was the png description.

**Comment by maryla-uc** [comment]
I agree with @kmilos that the best way would probably be to convert the metadata to XMP. However, I have a few concerns:

- It would increase the file size since XMP is quite verbose and stored uncompressed in AVIF
- There are alternatives available: use XMP in the first place if you really need this metadata
- This seems like a very niche use case. Other tools don't convert this metadata as far as we know?
- If there is already an XMP chunk, we would have to either merge the extra metadata with it, which is not entirely trivial (needs a dependency on an xml library and might raise new questions e.g. what if there is already a description in the XMP), or add an addtional XMP metadata box to the file, but not all readers would support that, in particular libavif only reads the first XMP box
- It feels out of scope to handle all various PNG keywords (see the "PNG TextualData Tags" table at https://exiftool.org/TagNames/PNG.html for the "known" ones, but a PNG file might also contian arbitrary keywords)

All in all, I think we can keep the issue open to see if there is more interest, but I would consider this low priority.
